lib/sentry_dart_plugin.dart

```diff
@@ -227,7 +232,13 @@ class SentryDartPlugin {
       List<String> releaseDartFilesParams = [];
       releaseDartFilesParams.addAll(params);
 
-      _addExtensionToParams(['dart'], releaseDartFilesParams, release, 'lib');
+      _addExtensionToParams(
+        ['dart'],
+        releaseDartFilesParams,
+        release,
+        'lib',
+        '~/lib/',
```

Comment:
do you know what happens for packages relying on other packages?

e.g I have a main app that consumes internal modules A and B 

I guess this would only cover uploading and prefixing files correctly for main app so frames coming from module A and B would not have source context?

Reply:
As long as the modules are within the lib folder, it "should" be fine. If they are outside, we are not supporting this currently, as we intentionally moved away from scanning the whole folder.

Reply:
I guess that can be a usecase though right? when they have internal libraries that they consume for example. we don't have to support it now but I guess we can create an issue and see if people are interested